src/screens/Main/RecordStack/QuestionsScreen/QuestionsScreen.tsx

```diff
@@ -23,17 +20,18 @@ interface Question {
 
 interface Props {
   navigation: NavigationScreenProp<NavigationState>,
+  videos: any,
```

Comment:
update to correct object type. `any` should really only be used for testing not permanently